### ISSUE TITLE
Import Email function not working with emails sourced from Outlook

### ISSUE THREAD

**Issue by alecisec** [gophish/gophish#2166]
What version of Gophish are you using?:
Have tried both 0.11.0 as well as building from source

Brief description of the issue:

What are you expecting to see happen? :
Email should import into the template when hitting the green "import" button after the red "import email" button

What are you seeing happen?
The email template remains blank, switching to source shows only

<title></title>
The console reports a malformed MIME header

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

have tried many, many different emails, all the same result "malformed MIME header"
Please provide any terminal output that may be relevant below:

```
time="2021-04-16T06:09:49Z" level=error msg="malformed MIME header: missing colon: %!q(MISSING)</style><![endif]--><style><!--"
```

the following email was used:
```html
<html xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns:w="urn:schemas-microsoft-com:office:word" xmlns:m="http://schemas.microsoft.com/office/2004/12/omml" xmlns="http://www.w3.org/TR/REC-html40"><head><meta name=Generator content="Microsoft Word 15 (filtered medium)"><style><!--
/* Font Definitions */
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
/* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0cm;
	margin-bottom:.0001pt;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	mso-fareast-language:EN-US;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:#0563C1;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-priority:99;
	color:#954F72;
	text-decoration:underline;}
span.EmailStyle17
	{mso-style-type:personal-compose;
	font-family:"Calibri",sans-serif;
	color:windowtext;}
.MsoChpDefault
	{mso-style-type:export-only;
	font-family:"Calibri",sans-serif;
	mso-fareast-language:EN-US;}
@page WordSection1
	{size:612.0pt 792.0pt;
	margin:72.0pt 72.0pt 72.0pt 72.0pt;}
div.WordSection1
	{page:WordSection1;}
--></style><!--[if gte mso 9]><xml>
<o:shapedefaults v:ext="edit" spidmax="1026" />
</xml><![endif]--><!--[if gte mso 9]><xml>
<o:shapelayout v:ext="edit">
<o:idmap v:ext="edit" data="1" />
</o:shapelayout></xml><![endif]--></head><body lang=EN-AU link="#0563C1" vlink="#954F72"><p class=msipheader6084391 align="Center" style="margin:0"><span style='font-size:12.0pt;font-family:Arial;color:#A80000'>OFFICIAL</span></p><br /><div class=WordSection1><p class=MsoNormal style='line-height:120%;text-autospace:none'>Signature<i><span lang=EN-US style='font-size:9.0pt;line-height:120%;font-family:"Arial",sans-serif;color:black'><o:p></o:p></span></i></p><p class=MsoNormal><o:p>&nbsp;</o:p></p></div></body></html>
```

**Comment by glennzw** [comment]
Going via the 'Import Email' I get the same error.

But pasting the HTML straight into the editor works for me:

![Screenshot 2021-04-20 at 11 15 59](https://user-images.githubusercontent.com/3966613/115379905-e789d380-a1c9-11eb-8a44-cc4e87d57cbd.png)

Although the render isn't great:

![Screenshot 2021-04-20 at 11 17 34](https://user-images.githubusercontent.com/3966613/115380020-0a1bec80-a1ca-11eb-9427-7518f9317e8a.png)

But the render appears the same in a browser, so I suspect gophish is doing what it should be doing.

Perhaps try this?

**Comment by glennzw** [comment]
I tested this from Outlook, and seems to work pretty well for me (importing the entire email including headers via `View Source`).

![Screenshot 2021-04-20 at 11 25 47](https://user-images.githubusercontent.com/3966613/115381212-3f750a00-a1cb-11eb-84fb-558876f6ff51.png)

**Comment by alecisec** [comment]
Perhaps it's Outlook version-dependent? I'm using Outlook's 'view source' option and have tried a large number of different emails without success. All gmail/hotmail etc. mail works fine however

**Comment by glennzw** [comment]
Can you paste a version here that's not working for me to test? I'm using Outlook on a Mac, maybe on Windows things are different.

**Comment by alecisec** [comment]
There's one in the original post as an example

**Comment by alecisec** [comment]
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"><p></p>
<p>Can you paste a version here that's not working for me to test? I'm using Outlook on a Mac, maybe on Windows things are different.</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">—<br>You are receiving this because you authored the thread.<br>Reply to this email directly, <a href="https://protect-au.mimecast.com/s/IJGgCyoNgQC6NOJPsZ5fHz?domain=github.com">view it on GitHub</a>, or <a href="https://protect-au.mimecast.com/s/IuirCzvOjVi8RWnYUXBxvU?domain=github.com">unsubscribe</a>.<img src="https://github.com/notifications/beacon/AGHEZOBLKFOP7SK2YWE6GRDTJ24MPA5CNFSM43A3ZRC2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOGEOUNPI.gif" height="1" width="1" alt=""></p>
<script type="application/ld+json">[
{
"@context": "https://protect-au.mimecast.com/s/mlaXCANZkjir9pVKF98Jd8?domain=schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://protect-au.mimecast.com/s/IJGgCyoNgQC6NOJPsZ5fHz?domain=github.com",
"url": "https://protect-au.mimecast.com/s/IJGgCyoNgQC6NOJPsZ5fHz?domain=github.com",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://protect-au.mimecast.com/s/PRAmCBNZ0liPVWAXijo8-e?domain=github.com"
}
}
]</script>

**Comment by glennzw** [comment]
If you're using the 'Import Email' function you need to include the headers, you're only importing the HTML. See my example again:

```
From: John Doe <jdoe@machine.example>
Sender: Michael Jones <mjones@machine.example>
To: Mary Smith <mary@example.net>
Subject: Saying Hello
Date: Fri, 21 Nov 1997 09:55:06 -0600
Message-ID: <1234@local.machine.example>

This is a message just to say hello.
So, "Hello".
```

**Comment by rajarshi-h** [comment]
Hi,

I am facing an issue. Opening outlook mail client in web, I first opened a mail, clicked on options, and then clicked on "View message details", thereafter I copied the entire text and pasted it in the "Import email" section of the gophish app. But I am getting an empty response to the "api/import/email" post api call as a result of which the html markdown editor is not being populated with any html/text.

Kindly help me out in figuring the issue.
